tests/test_settlements.py

```diff
@@ -0,0 +1,190 @@
+                                "fixed": {
+                                    "currency": "EUR",
+                                    "value": "0.2500"
+                                },
+                                "percentage": null
+                            },
+                            "amountNet": {
+                                "currency": "EUR",
+                                "value": "0.5000"
+                            },
+                            "amountVat": {
+                                "currency": "EUR",
+                                "value": "0.1050"
+                            },
+                            "amountGross": {
+                                "currency": "EUR",
+                                "value": "0.6050"
+                            }
+                        }
+                    ]
+                }
+            }
+        }"""
+    )
+
+    assert settlement.reference == '1234567.1804.03'
+    assert settlement.created_at == '2018-04-06T06:00:01.0Z'
+    assert settlement.settled_at == '2018-04-06T09:41:44.0Z'
+    assert settlement.amount == {'currency': 'EUR', 'value': '39.75'}
+    assert settlement.invoice_id == 'inv_FrvewDA3Pr'
+
+    # The following are not in the example json response.
+    # https://docs.mollie.com/reference/v2/settlements-api/get-settlement
+    assert settlement.status == settlement.STATUS_OPEN
+    assert settlement.is_open() is True
+    assert settlement.is_canceled() is False
+    assert settlement.is_failed() is False
+    assert settlement.is_pending() is False
+
+    assert settlement.chargebacks == chargebacks
+    assert settlement.payments == payments
+    assert settlement.refunds == refunds
+
+
+def test_settlement_get_next(client, response):
+    """Retrieve the details of the current settlement that has not yet been paid out."""
+    response.get('https://api.mollie.com/v2/settlements/next', 'settlement_next')
+
+    settlement = client.settlements.get('next')
+
+    assert isinstance(settlement, Settlement)
+    assert settlement.created_at == '2018-04-06T06:00:01.0Z'
+    assert settlement.settled_at is None
+    assert settlement.amount == {'currency': 'EUR', 'value': '39.75'}
+
+
+def test_settlement_get_open(client, response):
+    """Retrieve the details of the open balance of the organization. """
+    response.get('https://api.mollie.com/v2/settlements/open', 'settlement_open')
+
+    settlement = client.settlements.get('open')
+
+    assert isinstance(settlement, Settlement)
+    assert settlement.reference is None
+    assert settlement.created_at == '2018-04-06T06:00:01.0Z'
+    assert settlement.settled_at is None
+    assert settlement.amount == {'currency': 'EUR', 'value': '39.75'}
+
+
+@pytest.mark.parametrize(
+    "input,expected",
+    [
+        (None, IdentifierError),
+        ("foo", IdentifierError),
+        ("next", None),  # Valid
+        ("open", None),  # Valid
+        ("stl_", None),  # Valid
```

Comment:
Actually, this is not a valid example of a settlement ID. How about using the `SETTLEMENT_ID` constant you already have?

Reply:
done